code/modules/events/scrubber_clog.dm

```diff
@@ -0,0 +1,144 @@
+/datum/round_event_control/scrubber_clog
+	name = "Minor Scrubber Clog"
+	typepath = /datum/round_event/scrubber_clog
+	weight = 25
+	max_occurrences = 3
+	earliest_start = 5 MINUTES
+
+/datum/round_event/scrubber_clog
+	announceWhen = 1
+	startWhen = 10
+	endWhen = 6000
+	var/obj/machinery/atmospherics/components/unary/vent_scrubber/scrubber //Scrubber selected for the event
+	var/mob/spawned_mob = /mob/living/basic/cockroach //What mob will be spawned
+	var/severity = "Minor" //Severity of the event (how dangerous are the spawned mobs, and at what quantity)
+	var/maximum_spawns = 3 //Cap on the number of spawned mobs that can be alive at once
+	var/spawn_delay = 10 //Interval between mob spawns
+	var/list/living_mobs = list() //Used to track/limit produced mobs
+	var/clogged = TRUE //Used for tracking if the clog signal should be sent
```

Comment:
Convert these to auto-doc comments. 
Use three slashes `///`, and put them the line above.